plugins/importexport/onix30/filter/MonographONIX30XmlFilter.php

```diff
@@ -164,6 +163,9 @@ public function createProductNode(DOMDocument $doc, Submission $submission, Publ
         $deployment = $this->getDeployment();
         $context = $deployment->getContext();
 
+        PluginRegistry::loadCategory('importexport');
+        $plugin = PluginRegistry::getPlugin('importexport', 'Onix30ExportPlugin');
+
```

Comment:
Ah, I thought the plugin is an attribute of the deployment, like in the Crossref, DataCite, etc. plugins, but it seems for other import/export plugin this is not the case...